src/gc/transaction_level_gc_manager.cpp

```diff
@@ -116,14 +121,14 @@ int TransactionLevelGCManager::Unlink(const int &thread_id,
       break;
```

Comment:
`local_unlink_queue` is thread-local and will be updated by a single GC thread, while `unlink_queue` is not because worker thread can push to-be-recycled tuple versions into it.